### PR TITLE
feat(DLL): add dLL Plugin

### LINKED ISSUE #130: implement webpack dll
- [ ] create one bundle with all node_modules
- [ ] consume in the main package

for web package https://github.com/entria/entria-fullstack/tree/master/packages/web

### PR DESCRIPTION
**Pull request by PlayMa256**
fixes #130

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -66,12 +66,14 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
+    "prebuild": "yarn build:deps",
     "build": "cross-env NODE_ENV=production wp --config ./webpack.config",
     "jest:lint": "jest --config jest-eslint.config.js",
     "now-build": "cross-env NODE_ENV=production wp --config ./webpack.config",
     "relay": "relay-compiler --watchman false --src ./src --schema ../server/data/schema.graphql --language typescript",
     "start": "cross-env NODE_ENV=development wp --config ./webpack.config",
```
Comment (mauriciord) [inline packages/web/package.json]: not necessary in `prestart`?

Comment (PlayMa256) [reply packages/web/package.json]: i forgot it

File: packages/web/package.json [hunk 2]
```diff
@@ -66,12 +66,15 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
+    "prestart" "yarn build:deps",
```
Comment (jgcmarins) [inline packages/web/package.json]:
```suggestion
    "prestart": "yarn build:deps",
```

**Comment by jgcmarins** [comment]
ok, this error is because wp means webpack
just replaced it and worked \o/

**Comment by PlayMa256** [comment]
> ok, this error is because wp means webpack
> just replaced it and worked \o/

i've rerun the whole monorepo, i cleaned everything and installed it again, and it still worked... i'm not really sure how to reproduce that!

wp references webpack-nano, a small cli!

**Comment by sibelius** [comment]
I'm gonna close to wait for webpack v5

tks

**Pull request closed without merging**